static/js/markdown.js

```diff
@@ -237,7 +237,7 @@ exports.is_status_message = function (raw_content) {
 };
 
 function make_emoji_span(codepoint, title, alt_text) {
-    return '<span aria-label="' + title + '"' +
+    return '<span aria-label="' + title + '" id="message-emojis"' +
```

Comment:
Having multiple elements with the same id is invalid in HTML since the `id` must be unique, see https://html.spec.whatwg.org/multipage/dom.html#the-id-attribute. So this should probably be a `class` instead. I also find the plural `emojis` unfitting because it's just a single emoji.